src/main/java/com/iyzipay/model/PaymentCard.java

```diff
@@ -98,6 +109,7 @@ public String toString() {
                 .append("cardAlias", cardAlias)
                 .append("cardToken", cardToken)
                 .append("cardUserKey", cardUserKey)
+                .append("metadata", metadata)
```

Comment:
You should check payload string for map variables.

Reply:
it has the format `metadata={}` for empty map and `metadata={kkey=vvalue}` for a singleton map

Reply:
`metadata={kkey=vvalue, kkey2=vvalue2}` for full map =)